application/templates/cms/edit_dimension.html

```diff
@@ -46,8 +47,13 @@ <h2 class="heading-medium">Charts & Tables</h2>
                                            href="{{ url_for('cms.chartbuilder', topic_uri=topic.uri, subtopic_uri=subtopic.uri, measure_uri=measure.uri, version=measure.version, dimension_guid=dimension.guid) }}">
                                         {% if 'UPDATE' in  measure.available_actions() %}edit{% else %}
                                             view{% endif %}</a></td>
-                                    <td>{% if 'UPDATE' in  measure.available_actions() %}<a id="delete_chart"
-                                                                                            href="{{ url_for('cms.delete_chart', topic_uri=topic.uri, subtopic_uri=subtopic.uri, measure_uri=measure.uri, version=measure.version, dimension_guid=dimension.guid) }}">delete</a>{% endif %}
+                                    <td>
+                                    {% if 'UPDATE' in  measure.available_actions() %}
+                                        <form action="{{ url_for('cms.delete_chart', topic_uri=topic.uri, subtopic_uri=subtopic.uri, measure_uri=measure.uri, version=measure.version, dimension_guid=dimension.guid) }}" method="post">
+                                            {{ form.csrf_token() }}
```

Comment:
We don't seem to call `csrf_token` in any of our other references, so maybe we don't need to do it here either.

Reply:
How do you mean?  We have `{{ form.csrf_token() }}` in many places.

Reply:
Nowhere, outside of this branch, adds the `()`

Reply:
Good point, let's remove the empty brackets?

Reply:
Ah, sorry, I see what you mean.